src/python/ensembl/hive/HiveRESTClient.py

```diff
@@ -91,8 +91,8 @@ def fetch_input(self):
         Return response received.
         """
         with self._session_scope() as http:
-            self.response = http.request(method=self.param('method'),
-                                         url=self.param('endpoint'),
+            self.response = http.request(method=self.param_required('method'),
```

Comment:
Where can you find the API documentation BTW ? This is the source code https://github.com/psf/requests/blob/master/requests/sessions.py#L463-L506 and `method` has to be given since it calls `upper()` on it,